pkgs/data/fonts/source-han-serif/default.nix

```diff
@@ -0,0 +1,67 @@
+{stdenv, fetchurl, unzip}:
+
+let
+  makePackage = {variant, language, region, sha256Light, sha256Heavy}: stdenv.mkDerivation rec {
+    version = "1.004R";
+    name = "source-han-serif-${variant}-${version}";
+    revision = "f6cf97d92b22e7bd77e355a61fe549ae44b6de76";
+    urlPrefix = "https://github.com/adobe-fonts/source-han-serif/raw/${revision}/SubsetOTF/SourceHanSerif${region}";
+
+    buildInputs = [ unzip ];
+
+    srcs = [
+      (fetchurl {
+        url = "${urlPrefix}_EL-M.zip";
+        sha256 = sha256Light;
+      })
+      (fetchurl {
+        url = "${urlPrefix}_SB-H.zip";
+        sha256 = sha256Heavy;
+      })
+    ];
+
+    sourceRoot = ".";
+
+    installPhase = ''
+      mkdir -p $out/share/fonts/opentype
+      cp $( find . -name '*.otf' ) $out/share/fonts/opentype
+    '';
+
+    meta = {
+      description = "${language} subset of an open source Pan-CJK serif typeface";
+      homepage = https://github.com/adobe-fonts/source-han-sans;
+      license = stdenv.lib.licenses.ofl;
+      platforms = stdenv.lib.platforms.unix;
```

Comment:
Yes, I want to maintain this package and `source-han-sans`, but I don't have a commit right.